Sources/AWSXRayRecorder/Emitter.swift

```diff
@@ -5,7 +5,7 @@ public protocol XRayEmitter {
     func flush(_ callback: @escaping (Error?) -> Void)
 }
 
-public struct XRayNoopEmitter: XRayEmitter {
+public struct XRayNoOpEmitter: XRayEmitter {
```

Comment:
👍

Reply:
;-) previous PR was more juicy, in fact should have been divided int osmaller ones

the examples `gsoc-swift-tracing` look nice, let me follow up on that in https://github.com/slashmo/gsoc-swift-tracing/issues/36

Reply:
:-)

There's a WIP towards defining protocols for tracer instruments here (following open telemetry shape of API mostly): which may be of interest to you btw: https://github.com/slashmo/gsoc-swift-tracing/pull/62 

Thanks in advance, really looking forward to soon trying out xray with it the APIs.